src/makeInfinityAdder.js

```diff
@@ -5,6 +5,23 @@
  */
 function makeInfinityAdder() {
   // write code here
+  let result = 0;
+
+  const adder = (...args) => {
```

Comment:
think about naming this variable much better and understandable

Reply:
Fixed)

As I understood it is about `adder` variable, right?